MolecularNodes/load.py

```diff
@@ -167,11 +180,11 @@ def molecule_local(
     return mol_object
 
 
-def open_structure_rcsb(pdb_code, include_bonds = True):
+def open_structure_rcsb(pdb_code, cache_dir = Path(*config['default_cache_dir']).expanduser(), include_bonds = True):
```

Comment:
For this function, should the default be `cache_dir = None` to match the underlying `biotite` function? In the addon code we'll be supplying a location, for interfacing it via python though would a user expect it to be cached by default?